src/components/Home.js

```diff
@@ -10,8 +10,10 @@ const Home = () => {
     <div>
       <section className='px2 bg-blue-white'>
         <div className='py7 container relative'>
-          <h1 className='mt0 pb1 border-bottom'>National, state, and local crime data</h1>
-          <p className='m0 col-10 h3 serif'>
+          <h1 className='mt0 pb1 border-bottom border-red-bright'>
+            National, state, and local crime data
+          </h1>
+          <p className='m0 col-10 fs-b1 serif'>
```

Comment:
maybe `p` since these will generally be used for p elements, or we could just do `s` for just the general "size"